.env.sample

```
@@ -0,0 +1,10 @@
+DATABASE_URL = "rdbmsType://username:password@iphost:portNumber/databaseName"
+PORT = yourPortNumber
+JMT_SECRET = your JMT_SECRET KEY
+TYPEORM_CONNECTION = yourRebmsType
+TYPEORM_HOST = yourHostNumber
+TYPEORM_USERNAME = yourUserName
+TYPEORM_PASSWORD = yourPassWord
+TYPEORM_DATABASE = yourDbName
+TYPEORM_PORT = yourTypeOrmPortNumber
+TYPEORM_LOGGING = TRUE or FALSE
```

```suggestion
DATABASE_URL = "rdbmsType://username:password@iphost:portNumber/databaseName"

PORT = yourPortNumber

JMT_SECRET = your JMT_SECRET KEY

TYPEORM_CONNECTION = yourRebmsType
TYPEORM_HOST = yourHostNumber
TYPEORM_USERNAME = yourUserName
TYPEORM_PASSWORD = yourPassWord
TYPEORM_DATABASE = yourDbName
TYPEORM_PORT = yourTypeOrmPortNumber
TYPEORM_LOGGING = TRUE or FALSE
```

줄바꿈을 활용한 가독성 확보가 필요해보입니다~!